packages/bootstrap-4/package.json

```diff
@@ -17,7 +17,7 @@
     "cs-check": "prettier -l \"{src,test}/**/*.ts?(x)\"",
     "cs-format": "prettier \"{src,test}/**/*.ts?(x)\" --write",
     "lint": "eslint src test",
-    "precommit": "lint-staged",
```

Comment:
I'm curious about the removal of this?

Reply:
As mentioned [in the issue thread](https://github.com/rjsf-team/react-jsonschema-form/issues/3616#issuecomment-1559995974), this is not a real PR, just asking for validation on the generation direction:
> It's in no way polished, and in its current state breaks some other functionality, but I wanted to consult it with you to assess whether it's worth continuing on this route. Would you please look at it? It this roughly acceptable as a concept, or did you imagine something else?

This was just a hack to get Husky to let me push. Maybe there's a easier way to do this, I've never worked with it before, but this seemed like an obvious way to shut him up :D